src/parser/csql_grammar.y

```diff
@@ -3928,6 +3967,38 @@ alter_stmt
 			$$ = node;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
+		DBG_PRINT}}
+	| ALTER				/* 1 */
+		{ push_msg (MSGCAT_SYNTAX_SYNONYM_INVALID_ALTER); }	/* 2 */
+	  opt_access_modifier		/* 3 */
+	  SYNONYM			/* 4 */
+	  synonym_name			/* 5 */
+	  For				/* 6 */
+	  class_name			/* 7 */
+	  opt_comment_spec		/* 8 */
+		{ pop_msg(); }
+		{{
+
```

Comment:
Please add below
`DBG_TRACE_GRAMMAR(alter_stmt, | ALTER opt_access_modifier SYNONYM synonym_name For class_name opt_comment_spec);`

Reply:
Thank you. Added.